src/main/java/net/bramp/ffmpeg/FFmpegUtils.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Helper class with commonly used methods
  */
-public final class FFmpegUtils {
+public abstract class FFmpegUtils {
```

Comment:
If you can change this back to `public final class FFmpegUtils` I can accept the pull request.